### PR TITLE
Fix select reference not having a placeholder

### LINKED ISSUE #381: Bug: Reference plugin select input has no default option
Reference plugin select input has no default option. It just selects the first item in the list. 

## Steps for Reproducing

- Add/Edit a revision of content with a reference (Product)
- Go to the reference (Taxonomy - Status)
  - Observe that the first option is `General Availability (GA)` and not `-Select a Status-`.

## Screenshots

image

## Affected Browsers

- [ ] All

## Runtime Version

5.11

## Code Version

2.10.0

### PR DESCRIPTION
**Pull request by Snugug**
Adds default select option for reference

---
Resolves #381 

`DCO 1.1 Signed-off-by: Sam Richard <sam@snug.ug>`

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -327,6 +327,11 @@ const references = (types) => {
  */
 const fill = (types, type, refs, database) => {
   const ct = _.cloneDeep(type);
+  const placeholder = {
+    value: '',
+    label: '-- Choose One --',
```
Comment (scottnath) [inline lib/utils.js]: There should be an optional way to change this text. Can you please add a [setting option for references](https://github.com/punchcard-cms/punchcard/blob/master/input-plugins/input-plugin-reference/index.js#L32) with this text as default?

Comment (Snugug) [inline lib/utils.js]: Yup!

Comment (Snugug) [inline lib/utils.js]: @scottnath Take a look, updated

File: input-plugins/input-plugin-reference/index.js [hunk 2]
```diff
@@ -31,9 +31,10 @@ module.exports = {
       options: [],
       settings: {
         empty: false,
```
Comment (scottnath) [inline input-plugins/input-plugin-reference/index.js]: can you delete this line while in here please?

**Comment by scottnath** [comment]
@Snugug small thing, then GTG!

File: input-plugins/input-plugin-reference/index.js (hunk 2 above)
Comment (Snugug) [inline input-plugins/input-plugin-reference/index.js]: THAT'S OUT OF SCOPE!
BUT THAT'S OK

Comment (Snugug) [inline input-plugins/input-plugin-reference/index.js]: While I'm in there, should I write validation to ensure that the item chosen is one of the options selected? Or should we write an actual story for that?

Comment (scottnath) [inline input-plugins/input-plugin-reference/index.js]: I think a story for that please! I think there is more expansion on this plugin than just that, so let that part stay OOScope for now...but not my `remove empty` request...that's the good kind of OOScope - ie very smartypants.

**Pull request merged**